_posts/2021-07-06-Influential-tech-lead.md

```diff
@@ -0,0 +1,90 @@
+At that point I had received very little formal managerial training, so learning on the job or from mentors was crucial to be successful.
+The soft skills I needed for my new role turned out to be more difficult than I initially thought.
+One of those difficult soft skills for me was (and often still is) the ability to influence people.
+Influencing others is not just about getting them to agree with your point of view or manipulating them to get your way, nor does it involve forcing others to change by using power and control.
+It's about noticing what motivates others and using that knowledge to leverage performance and positive results.
+
+Taking up the technical team lead role is not always an easy one.
+Often the skills necessary to be a great developer don't translate easily to the role of a technical team lead.
+In this post, I will give some tips on how to be an influential technical team lead.
+
+## Have a clear vision
+A vision is a clear image of how you see the future.
+It’s something that keeps you motivated and excited to do what you do.
+A technical team lead must be able to both zoom in and zoom out on a project.
+On one hand, you need to be able to look in detail into a technical requirement.
+On the other hand, you also need to keep the bigger picture in mind and how the requirement fits into the greater vision.
+If you're a technical team lead that has no vision and isn't going anywhere, then how can you expect your team will follow you?
+In order to create a vision you can ask yourself questions like "What do you want to achieve?, "Which (tangible) output do you want when achieving your goal?", "How would that output change the way things are done now?", ...
+Setting specific goals together with your team that move towards your vision will help you to achieve it.
+
+Once you have that vision and goals, being predictable is key.
+By doing so, everybody knows what you're trying to achieve.
+The Don’t-Repeat-Yourself principle, which is a key principle for each software engineer, does not apply when working with people.
+Communicate your vision and goals over and over again: repetition and predictability are key.
+
+## Be collaborative
+A top-down approach where nobody has a say in the process and where both vision and goals are pushed downwards, is very easy for team members to reject.
+A different and more sustainable approach is to be a collaborative technical team lead.
+This means that you regularly seek out a diversity of opinions and ideas amongst teammates to solve problems.
+Involve your team to create a shared vision, and identify the behaviours necessary to accomplish it.
+It's all about getting the team to think through these difficulties themselves instead of telling them what to do all the time.
+As a result they are more engaged, feel trusted, and are more likely to take ownership of their work.
+Building meaningful relationships is one of the key responsibilities of a technical team lead.
+
```

Comment:
I would also add something about taking the time on the WHY. Taking the time to get to understand the why of a different approach, why a team member proposes a different approach. Only when that is clear can choices as a team be made.

Reply:
Good point, but I think this is also covered more in depth in the explanation of Icek Ajzen's model